packages/Calendar/styles.js

```diff
@@ -0,0 +1,197 @@
+import styled from 'styled-components'
+
+const theme = {
+  colors: {
```

Comment:
These colours can now be imported using `@tds/core-colours@1.1.1` - https://tds.telus.com/components/index.html#colours